src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseActionsResource.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.digital.ho.hocs.casework.api;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
Is this redundant... looks to have been added but not implemented anywhere

Reply:
Yes. I have now removed.